spec/system/page_objects/components/d_page_header.rb

```diff
@@ -14,6 +22,20 @@ def visible?
       def hidden?
         has_no_css?(".d-page-header")
       end
+
+      private
+
+      def tabs_container_selector
+        "ul.d-nav-submenu__tabs"
+      end
+
+      def tab_item_selector(tab_name)
+        "li[class$='-tabs__#{tab_name}']"
```

Comment:
I don't think we benefit from having the page name in each individual tab element, e.g. `admin-backups-tabs__` etc. but we can address that later. For now using this "end of string" CSS matcher to skip over it.